Hameg_8123App/devHameg_8123.proto

```diff
@@ -0,0 +1,105 @@
+
+
+Terminator = CR;
+
+getIDN {
+    out "IDN";
+    in "%39c";
+}
+
+startCounting {
+    out "STR";
+}
+
+stopCounting {
+    out "STP";
+}
+
+getCounts {
+    out "XMT";
+    # Responds with value, units
+    in "%d, %(\$1)39c";
```

Comment:
`.EGU` is `EGU | Engineering Units | STRING [16]`, probably a bit of an edge case if the device responds with a long unit string but maybe make this `%(\$1)16c`